sdk/digitaltwins/Azure.DigitalTwins.Core/README.md

```diff
@@ -57,8 +69,24 @@ Serialization helpers provided to help serialize/deseralize commonly used types
 Any time the client library code is updated, you will need to run the script [here](./../Export-AdtApis.ps1), which will update the corresponding API surface.
 
 ## Troubleshooting
+All Azure Digital Twin service operations will throw a RequestFailedException on failure with helpful ErrorCodes.
+
+For example, if you use the ```GetModelAsync``` operation and the model you are looking for doesn't exist, you can catch that specific [HttpStatusCode][http_status_code] to decide the operation that follows in that case.
+```csharp
```

Comment:
Can you tie this to a snippet that gets updated automatically?

Reply:
I didn't want to put too much in here, so I just kept it as simple as possible, I think our samples should have enough stuff in them so we don't have to keep updating this, thoughts?